src/components/ListItem/ListItem.vue

```diff
@@ -187,7 +191,9 @@
 		<a :id="anchorId"
 			ref="list-item"
 			:class="{ 'list-item--active' : active }"
-			href="#"
+			:href="href"
+			target="_blank"
+			rel="noopener noreferrer"
```

Comment:
These two now always apply! This breaks the component.

Reply:
Shouldn't it be harmless on a `<a href="#">` tag? How does it break it?

Reply:
Yes, we use @click for it, no?

Reply:
Yes, but the target blank seems to kick in before click is emitted. Thus the appnavigation on forms now opens a new tab with the link instead of triggering the router.

Reply:
https://github.com/nextcloud/nextcloud-vue/pull/3069 should do it. Click event would have a different behaviour if `e.preventDefault()` wouldn't be used.